### DISCUSSION TITLE
combo number sometimes leaves slider head in editor

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21132] [Q&A] [answered]
when you seek in the editor timeline while in on the slider head, sometimes the combo # will come off the slider head with the follow point.

bugged behavior:
![image](https://user-images.githubusercontent.com/78441708/200102054-699a887d-8cd4-4d76-b875-8b89dd6a6057.png)

intended behavior
![image](https://user-images.githubusercontent.com/78441708/200102117-e77b42f1-c373-42e3-9d1b-4d4bfb55397e.png)

**Comment by ghost** [comment]
this is intended

https://github.com/ppy/osu/wiki/Gameplay-differences-from-osu!stable#slider-head-circle-follows-track-snaking-when-not-hit

**Reply by Axilotl17** [reply]
This is different behavior. look at the 2 screenshots in my original post. they should be the same, but there is a 1 on the first screenshot on the follow point.

**Reply by ghost** [reply]
This appears to follow the behaviour in the wiki article linked

Are you suggesting the slider head stay where it is instead of moving? I don't understand your post (you mention followpoints but I don't see a followpoint in either screenshot. Maybe you meant the sliderball or follow circle?)

**Reply by Axilotl17** [reply]
what im saying is the two screenshots should be the same?!?! also yes i mean sliderball/follow circle.
if you look, in the first screenshot, there is a faint 1 on the sliderball.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/10909
